cve_bin_tool/cvedb.py

```diff
@@ -531,11 +525,41 @@ def populate_severity(self, severity_data, cursor, data_source):
             except Exception as e:
                 LOGGER.info(f"Unable to insert data for {data_source} - {e}\n{cve}")
 
+    def populate_cve_metrics(self, severity_data, cursor):
+        insert_cve_metrics = self.INSERT_QUERIES["insert_cve_metrics"]
+        del_cve_range = "DELETE from cve_range where CVE_number=?"
+
+        for cve in severity_data:
+            # Check no None values
+            if not bool(cve.get("score")):
+                LOGGER.debug(f"Update score for {cve['ID']}")
+                cve["score"] = "unknown"
+            if not bool(cve.get("CVSS_version")):
+                LOGGER.debug(f"Update CVSS version for {cve['ID']}")
+                cve["CVSS_version"] = "unknown"
+            if not bool(cve.get("CVSS_vector")):
+                LOGGER.debug(f"Update CVSS Vector for {cve['ID']}")
+                cve["CVSS_vector"] = "unknown"
+
+        for cve in severity_data:
+            try:
+                cursor.execute(
+                    insert_cve_metrics,
+                    [
+                        cve["ID"],
+                        cve["CVSS_version"],
```

Comment:
The second parameter of the cve_metrics table should be referencing the name of the metric in the metrics table and not be the metric name

For a CVSS-3 CVE, we should have CVE_number, 3, <score>, <vector> where 3 is the INDEX of the CVSS-3 string in the metrics table

Reply:
Ok sure, I will update it.